projects/m2/023-scrabble-score/solutions/enricozoboli/solution_scrabble_score.py

```diff
@@ -0,0 +1,38 @@
+char_point = { 
+    1 : ['a','e', 'i', 'l', 'n', 'o', 'r', 's', 't', 'u'],
```

Comment:
Perhaps using chars as keys (ex. 'aeilnorstu': 1 ) and then search the char with an `in` method would be a quicker solution